rep-0003.rst

```diff
@@ -169,14 +200,12 @@ These targets, including starting and ending support dates, are based on the Dis
 C++
 ---
 
-We use the C++03 (ISO/IEC 14882:2003) standard, and are compiler-agnostic.
+As of January 2015 (Hydro, Indigo, and Jade), we use the C++03 (ISO/IEC 14882:2003) standard, and are compiler-agnostic.
```

Comment:
What I wanted to do was to make sure people in the far future see when this applied in case we forget to update it in future iterations, which is something that seems to happen a lot. I can try to reword it to be clear. Maybe:

> As of January 2015 (Hydro, Indigo, and Jade), we are still using ...

I can also just remove it, I'm agnostic.